glib/src/functions.rs

```diff
@@ -283,6 +283,7 @@ pub fn file_open_tmp(
 ///
 /// This can be called from any thread and will execute the future from the thread
 /// where main context is running, e.g. via a `MainLoop`.
+#[cfg(feature = "futures")]
 pub fn spawn_future<R: Send + 'static, F: std::future::Future<Output = R> + Send + 'static>(
```

Comment:
How do these functions depend on the futures crate? They should work with just `std`

Reply:
right